### ISSUE TITLE
Clinicians - see last updated tag on Meds

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/oxleas-adhd#319]

**Comment by iteles** [comment]
@katyedwards Is this simply a line saying ‘Last updated on ...’ at either top or bottom of the page?

Will this include, date, date and time and/or who performed the latest update?

**Comment by ghost** [comment]
@iteles yes, the tag exists for SUs, but the clinician view doesn't see it.  I'll add an example to the description...
